src/utils.cpp

```diff
@@ -264,25 +299,28 @@ void ZypperUtils::initTarget( const Pathname& sysRoot )
 
 void ZypperUtils::makeNecessaryChangesToRunAsRoot( const Pathname& sysRoot )
 {
-    /* Caller: ZypperUtils::install()
-     * Can be absolutely certain that s_repoManagerOpts, and s_repoManager
-     * are already intialized [to display information]
+    /* Need to remove temporary PoolItems (loaded into pool when querying package information (metadata))
+     * from the ResPool to avoid unnecessary conflicts.
+     * 
+     * The following code snippet does the work but needs refinement
+     * if ( s_zypp->pool().empty() ) {
+     * 	   Repository repo = s_zypp->pool().reposFind( "temp" );
+     *     repo.eraseFromPool();
+     * }
+     * 
+     * Reminder To Me (Shalom): Investigate reposEraseAll() [sat/Pool.h]
```

Comment:
You can call `sat::Pool::instance().reposEraseAll() ` but then you should  '(Build)Require: libzypp >=16.1.1'.  If you intend to make your code available to older openSUSE versions as well (e.g. Leap-42.1 / libzypp 15.x), you should not use it.